h/browser/chrome/lib/options.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hypothesis Extension</title>
```

Comment:
Small nit: Even though this is not visible to the user, I'd suggest setting this to "Hypothesis Extension Settings" so that it becomes more obvious which page is being referred to if this shows up in the Task Manager etc.